numpy/lib/tests/test_function_base.py

```diff
@@ -2023,6 +2023,15 @@ def test_extreme(self):
         assert_array_almost_equal(c, np.array([[1., -1.], [-1., 1.]]))
         assert_(np.all(np.abs(c) <= 1.0))
 
+    @pytest.mark.parametrize("test_type", [getattr(np, dtype, None) for dtype \
+    in ['float32', 'float64', 'float96', 'float128']])
+    def test_cov_dtype(self, test_type):
+        if test_type is None:
+            pytest.skip("Data type not available")
```

Comment:
Rather than skipping, just use the names which always exist:
```suggestion
    @pytest.mark.parametrize("test_type", [np.half, np.single, np.double, np.longdouble])
    def test_cov_dtype(self, test_type):
```

Reply:
Nice, I wasnt aware of those. Thats much better, thanks!

Reply:
The docs aren't clear about that yet, so it's understandable you missed them - I'm updating them in #17331. The link at the top of that PR will take you to the new version.